tests/Microsoft.Identity.Test.Unit/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -691,6 +686,97 @@ public async Task GetAuthorizationRequestUrlNoRedirectUriTestAsync()
             }
         }
 
+        [TestMethod]
+        public async Task GetAuthorizationRequestUrl_IgnoreLoginHint_UseCcsRoutingHint_TestAsync()
+        {
+            using (var httpManager = new MockHttpManager())
+            {
+                ConfidentialClientApplication app = CreateCca(httpManager);
+
+                var uri = await app
+                    .GetAuthorizationRequestUrl(TestConstants.s_scope)
+                    .WithLoginHint(TestConstants.DisplayableId)
+                    .WithCcsRoutingHint("oid", "tid")
```

Comment:
Should add to the method comment that routing hint will take precedence over login hint.